src/site/markdown/sqlmap-xml.md

```diff
@@ -0,0 +1,1483 @@
+
+The next sections will walk through each of the elements in more detail.
+
+#### id & result
+
+```xml
+<id property="id" column="post_id"/>
+<result property="subject" column="post_subject"/>
+```
+
+These are the most basic of result mappings. Both *id* and *result* map a single column value to a single property or field of a simple data type (String, int, double, Date, etc.).
+
+The only difference between the two is that *id* will flag the result as an identifier property to be used when comparing object instances. This helps to improve general performance, but especially performance of caching and nested result mapping (i.e. join mapping).
+
+Each has a number of attributes:
+
+| Attribute     | Description                                                  |
+| ------------ | ------------------------------------------------------------ |
+| `property`    | The field or property to map the column result to. If a matching JavaBeans property exists for the given name, then that will be used. Otherwise, MyBatis will look for a field of the given name. In both cases you can use complex property navigation using the usual dot notation. For example, you can map to something simple like: `username`, or to something more complicated like: `address.street.number`. |
+| `column`      | The column name from the database, or the aliased column label. This is the same string that would normally be passed to `resultSet.getString(columnName)`. |
+| `javaType`    | A fully qualified Java class name, or a type alias (see the table above for the list of built-in type aliases). MyBatis can usually figure out the type if you're mapping to a JavaBean. However, if you are mapping to a HashMap, then you should specify the javaType explicitly to ensure the desired behaviour. |
+| `jdbcType`    | The JDBC Type from the list of supported types that follows this table. The JDBC type is only required for nullable columns upon insert, update or delete. This is a JDBC requirement, not a MyBatis one. So even if you were coding JDBC directly, you'd need to specify this type – but only for nullable values. |
+| `typeHandler` | We discussed default type handlers previously in this documentation. Using this property you can override the default type handler on a mapping-by-mapping basis. The value is either a fully qualified class name of a TypeHandler implementation, or a type alias. |
+[Id and Result Attributes]
+
+#### Supported JDBC Types
+
+For future reference, MyBatis supports the following JDBC Types via the included JdbcType enumeration.
+
+|            |           |               |                 |           |             |
```

Comment:
@harawata It seems that we can't have a table which has no head in Markdown.

Reply:
@awxiaoxian2020 ,

Try removing the header line.

```suggestion
```